fecgov/fec-cms
# 5215 house and senate overview totals

## Summary

- Resolves #5215 

Quite a few changes to [/data/elections/house/](http://127.0.0.1:8000/data/elections/house/) and [/data/elections/senate/](http://127.0.0.1:8000/data/elections/senate/)

### Required reviewers

Front-end and UX?

## Impacted areas of the application

The House & Senate overview pages

## Screenshots

Before:
![image](https://user-images.githubusercontent.com/26720877/196201798-dd459d04-508e-4135-bc7e-209ba7f4bee6.png)

This PR:
![image](https://user-images.githubusercontent.com/26720877/196201906-38e047b2-ae48-457c-ae56-768909514eb9.png)


## Related PRs

None

## Known TODOs

- Look & feel of the summary section (and maybe the others)
- Initial data and load appearance/content
- Values and formulas verification (Paul?)
- Layouts for the in-between sizes
- Organization of the rest of the page—section titles, tab organization, flags
- Specific TODOs in the code—features, etc
- Clean up the fetch URL, specifically add the api_key
- Testing w/ non-Chromium browsers

## How to test

- pull the branch
- `npm i`
- `npm run build`
- `./manage.py runserver`
- [House elections page](http://127.0.0.1:8000/data/elections/house/)
   - initial load works?
   - changing the election year pull-down works?
   - changing tabs works?
   - changing tabs or years, copy the URL, paste it back in. Does it work as expected?
   - are the other tabs' content broken?
- [Senate elections page](http://127.0.0.1:8000/data/elections/senate/)
   - initial load works?
   - changing the election year pull-down works?
   - changing tabs works?
   - changing tabs or years, copy the URL, paste it back in. Does it work as expected?
   - are the other tabs' content broken?
